### PR TITLE
🤗NER task support in Huggingface dataset

### PR DESCRIPTION
**Pull request by trajepl**
## Describe your changes
1. Add support to NER task dataset.
2. Attach the test config file for NER model

## Checklist before requesting a review
- [ ] Add unit tests for this change.
- [ ] Make sure all tests can pass.
- [ ] Update documents if necessary.
- [ ] Format your code by running `pre-commit run --all-files`
- [ ] Is this a user-facing change? If yes, give a description of this change to be included in the release notes.

### CODE REVIEW

**Review by lainey1570 — APPROVED** [review]
The PR looks good and it would meet my requirements for `ner`.

File: olive/data/component/post_process_data.py [hunk 1]
```diff
@@ -30,3 +30,22 @@ def text_classification_post_process(_output_data, **kwargs):
         _, preds = torch.max(_output_data, dim=1)
 
     return preds
+
+
+@Registry.register_post_process()
+def ner_post_process(_output_data, **kwargs):
```
Comment (xiaoyu-work) [inline olive/data/component/post_process_data.py:36]: Any documentations need to be updated about how to use this?

Comment (trajepl) [reply olive/data/component/post_process_data.py:36]: https://github.com/microsoft/Olive/pull/401/files#diff-fce744f2268555668fc1868242a7833b6691a78bc883d9dbfc50fcc23f70a0b9

Here I add the template config file for `ner` task. This function will not be exposed to user directly now. When user set the task as ner, the corresponding pre_process, post_process will be picked.

Comment (trajepl) [reply olive/data/component/post_process_data.py:36]: https://github.com/microsoft/Olive/blob/main/docs/architecture.md#data-container Here is the doc to use that function internally now.

**Review by lainey1570 — APPROVED** [review]

**Pull request merged**